examples/multiple_windows/lib/app/regular_window_content.dart

```diff
@@ -0,0 +1,149 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: invalid_use_of_internal_member
+// ignore_for_file: implementation_imports
+
+import 'package:flutter/material.dart';
+import 'window_content.dart';
+import 'models.dart';
+import 'rotated_wire_cube.dart';
+import 'dart:math';
+import 'package:flutter/src/widgets/_window.dart';
+
+class RegularWindowContent extends StatefulWidget {
+  const RegularWindowContent({super.key, required this.window});
+
+  final RegularWindowController window;
+
+  @override
+  State<StatefulWidget> createState() => _RegularWindowContentState();
+}
+
+class CallbackRegularWindowControllerDelegate
+    with RegularWindowControllerDelegate {
+  CallbackRegularWindowControllerDelegate({required this.onDestroyed});
+
+  @override
+  void onWindowDestroyed() {
+    onDestroyed();
+    super.onWindowDestroyed();
+  }
+
+  final VoidCallback onDestroyed;
+}
+
+class _RegularWindowContentState extends State<RegularWindowContent>
+    with SingleTickerProviderStateMixin {
+  late final AnimationController _animation;
```

Comment:
I would handle all of the animation stuff inside RotatedWireCube because it's an implementation detail of that widget.